<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_TimelineHeaderWidget</class>
 <widget class="QFrame" name="Frame_TimelineHeaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QDockWidget" name="dockWidget_Main">
     <property name="features">
      <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::LeftDockWidgetArea</set>
     </property>
     <property name="windowTitle">
      <string>Track 0</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lblTitle">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser_Main">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::VLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="documentTitle">
            <string>test</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
           <property name="openLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
           <item>
            <widget class="QToolButton" name="toolButton_0">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_3">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
